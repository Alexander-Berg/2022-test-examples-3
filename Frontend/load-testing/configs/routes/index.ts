/* eslint-disable */
const trimLeadingSlash = (str: string) => str.replace(/^\/+/, '');

const skillRouteConstructor = (prefix: string) => {
    const prefixedRouteBuilder = (route: string) => (skillId: string) =>
        `${prefix}/skills/${skillId}/${trimLeadingSlash(route)}`;

    return {
        'dev-console/v1/skills/:skillId/draft': prefixedRouteBuilder('draft'),
        'dev-console/v1/skills/:skillId/draft/actions/withdraw': prefixedRouteBuilder('draft/actions/withdraw'),
        'dev-console/v1/skills/:skillId/flags': prefixedRouteBuilder('flags'),
        'dev-console/v1/skills/notifications/options': prefixedRouteBuilder('notifications/options'),
        'dev-console/v1/skills/notifications/phone': prefixedRouteBuilder('notifications/phone'),
        'dev-console/v1/skills/notifications/phone/confirmation': prefixedRouteBuilder(
            'notifications/phone/confirmation',
        ),
        'dev-console/v1/skills/candidate': prefixedRouteBuilder('candidate'),
        'dev-console/v1/skills/release': prefixedRouteBuilder('release'),
        'dev-console/v1/skills/message': prefixedRouteBuilder('message'),
        'dev-console/v1/skills/operations': prefixedRouteBuilder('operations'),
        'dev-console/v1/skills/reviews': prefixedRouteBuilder('reviews'),
        'dev-console/v1/skills/logo': prefixedRouteBuilder('logo'),
        'dev-console/v1/skills/actions/stop': prefixedRouteBuilder('actions/stop'),
    };
};

const makeConsoleRoutes = () => {
    const prefix = '/api/dev-console/v1';
    const prefixed = (route: string) => `${prefix}/${trimLeadingSlash(route)}`;

    return {
        'dev-console/v1/skills': prefixed('skills'),
        'dev-console/v1/skills/:skillId': (skillId: string) => prefixed(`skills/${skillId}`),
        'dev-console/v1/snapshot': prefixed('snapshot'),
        'dev-console/v1/organizations': prefixed('organizations'),
        'dev-console/v1/organizations/:orgId': (orgId: string) => prefixed(`organizations/${orgId}`),
        'dev-console/v1/subscription': prefixed('subscription'),
        'dev-console/v1/email': prefixed('email'),
        'dev-console/v1/email/send': prefixed('email/send'),
        'dev-console/v1/moderation/skills/:skillId/validate-token': (skillId: string) =>
            prefixed(`moderation/skills/${skillId}/validate-token`),
        'dev-console/v1/oauth/apps': prefixed('oauth/apps'),
        'dev-console/v1/oauth/apps/:appId': (appId: string) => prefixed(`oauth/apps/${appId}`),
        ...skillRouteConstructor(prefix),
    };
};

const makeCatalogueRoutes = () => {
    const prefixed = (route: string) => `/api/catalogue/v1/${trimLeadingSlash(route)}`;

    return {
        'catalogue/v1/categories': prefixed('categories'),
        'catalogue/v1/dialogs/:slug': (slug: string) => prefixed(`dialogs/${slug}`),
        'catalogue/v1/dialogs/search': prefixed('dialogs/search'),
        'catalogue/v1/dialogs/suggest': prefixed('dialogs/suggest'),
        'catalogue/v1/dialogs2': prefixed('dialogs2'),
        'catalogue/v1/dialogs/:skillId/slug': (skillId: string) => prefixed(`dialogs/${skillId}/slug`),
        'catalogue/v1/dialogs/:skillId/account_linking': (skillId: string) =>
            prefixed(`dialogs/${skillId}/account_linking`),
        'catalogue/v1/collections': prefixed('collections'),
        'catalogue/v1/builtin-collections': prefixed('builtin-collections'),
        'catalogue/v1/reviewform': prefixed('reviewform'),
        'catalogue/v1/experiments/:yandexuid': (yandexuid: string) => prefixed(`experiments/${yandexuid}`),
        'catalogue/v1/compilations/:slug': (slug: string) => prefixed(`compilations/${slug}`),
        'catalogue/v1/pages/main': prefixed('pages/main'),
        'catalogue/v1/smart_home/get_native_skills': prefixed('smart_home/get_native_skills'),
    };
};

const makeExternalRoutesV1 = () => {
    return {
        'external/v1': '/api/external/v1',
    };
};

const makeExternalRoutesV2 = () => {
    const prefixed = (route: string) => `/api/external/v2/${trimLeadingSlash(route)}`;

    return {
        'external/v2/skills/:skillId': (skillId: string) => prefixed(`skills/${skillId}`),
        'external/v2/chats/bulk/get': prefixed('/chats/bulk/get'),
        'external/v2/skills/bulk/get': prefixed('/skills/bulk/get'),
        'external/v2/chats': prefixed('chats'),
        'external/v2/chats/:skillId': (skillId: string) => prefixed(`chats/${skillId}`),
        'external/v2/chats/:skillId/draft': (skillId: string) => prefixed(`/chats/${skillId}/draft`),
        'external/v2/chats/:skillId/draft/status': (skillId: string) => prefixed(`/chats/${skillId}/draft/status`),
        'external/v2/chats/:skillId/draft/actions/deploy': (skillId: string) =>
            prefixed(`/chats/${skillId}/draft/actions/deploy`),
    };
};

const makePublicRoutes = () => {
    const prefixed = (route: string) => `/api/public/v1/${trimLeadingSlash(route)}`;

    return {
        'public/v1/status': prefixed('status'),
        'public/v1/skills/:skillId/images': (skillId: string) => prefixed(`skills/${skillId}/images`),
        'public/v1/skills/:skillId/images/:groupId/:imageName': (skillId: string, groupId: string, imageName: string) =>
            prefixed(`/skills/${skillId}/images/${groupId}/${imageName}`),
        'public/v1/skills/:skillId/sounds': (skillId: string) => prefixed(`skills/${skillId}/sounds`),
        'public/v1/skills/:skillId/sounds/:soundId': (skillId: string, soundId: string) =>
            prefixed(`skills/${skillId}/sounds/${soundId}`),
    };
};

const routesConfig = {
    ...makeConsoleRoutes(),
    ...makeCatalogueRoutes(),
    ...makePublicRoutes(),
    ...makeExternalRoutesV1(),
    ...makeExternalRoutesV2(),
};

export default routesConfig;

export type RoutesStructure<V> = { [K in keyof typeof routesConfig]: V };
export type PartialRoutesRecord<V> = Partial<RoutesStructure<V>>;
