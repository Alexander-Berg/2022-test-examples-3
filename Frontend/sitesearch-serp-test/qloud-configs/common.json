{
    "qloud": {
        "host": "platform.yandex-team.ru",
        "id": "sitesearch.sitesearch-serp-test.common",
        "domains": [
            {
                "type": "common",
                "domainName": "sitesearch.common.yandex.net"
            }
        ],
        "component": {
            "name": "nginx-app",
            "image": {
                "repo": "{{DOCKER_IMAGE_REPO}}",
                "tag": "{{DOCKER_TAG}}"
            },
            "network": "WMFRONT_TESTING_NETS",
            "routes": ["/"]
        }
    }
}
