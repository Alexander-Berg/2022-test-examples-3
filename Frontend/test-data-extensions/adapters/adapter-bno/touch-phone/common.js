module.exports = function(app, raiffeisen, meta) {
    return {
        type: 'snippet',
        data_stub: function(dataBuilder) {
            return {
                num: 0,
                construct: dataBuilder.create()
                    .setAdapter('test')
                    .setTemplate('bno')
                    .add(raiffeisen)
                    .add(app)
                    .add(meta)
                    .build()
            };
        }
    };
};
