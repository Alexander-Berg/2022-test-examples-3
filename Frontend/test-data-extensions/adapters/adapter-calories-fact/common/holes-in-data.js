var dataStub = require('../data')();

dataStub.construct.data.size = [
    {
        text_i18n: 'calories_100g'
    },
    {
        value: 17
    },
    {
        value: 5,
        text_i18n: 'teaspoon'
    }
];

dataStub.construct.data.attributes = [
    {
        value: 0.52
    },
    {

        text_i18n: 'fat'
    },
    {
        value: 9.8,
        text_i18n: 'carbohydrate'
    }
];

module.exports = {
    type: 'snippet',
    data_stub: dataStub
};
