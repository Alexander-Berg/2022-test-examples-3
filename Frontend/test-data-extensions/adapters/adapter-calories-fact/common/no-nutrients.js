var dataStub = require('../data')();

dataStub.construct.data.attributes = [];

module.exports = {
    type: 'snippet',
    data_stub: dataStub
};
