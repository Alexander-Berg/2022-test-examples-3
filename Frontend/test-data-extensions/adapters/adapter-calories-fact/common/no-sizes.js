var dataStub = require('../data')();

dataStub.construct.data.size = [];

module.exports = {
    type: 'snippet',
    data_stub: dataStub
};
