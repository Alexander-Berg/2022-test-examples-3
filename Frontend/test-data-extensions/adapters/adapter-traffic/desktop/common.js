'use strict';

module.exports = function(opts) {
    return {
        type: 'snippet',
        extensions: {
            time: '2017-03-07T14:44:41+0300'
        },
        data_stub: {
            base_type: null,
            basehost: null,
            by_link: '',
            doc: 'YxWeb::Search::Doc: YxWeb::Search::Doc=HASH(0x6375a708)',
            docid: '14-',
            docindgen: 0,
            doctitle: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
            fake_doc: '',
            favicon_domain: 'yandex.ru',
            geo: null,
            green_url: 'https://yandex.ru/…/probki?source=traffic',
            group_count: 1,
            group_relevance: -1,
            has_html5_video_player: null,
            headline: '',
            headline_relevant: '',
            host: 'yandex.ru',
            host_id: 'yandex.ru/maps/213/moscow/probki?source=traffic',
            is_index_page: 0,
            is_recent: '1',
            lang: null,
            markers: {
                FakeHash: '15388277386232187162',
                Rule: 'Vertical/traffic',
                Slices: 'TRAFFIC:20.0|',
                WizardPos: '0'
            },
            metahosts: ['ws33-124.search.yandex.net:9080'],
            mime: '',
            mnem_prio: 'fuzzy',
            mtime: 1488887299,
            noarchive: null,
            num: '0',
            passages: [],
            priority: null,
            relevance: '-1',
            relevdata: {
                fixurl: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
                given: 0
            },
            reqans_fields: {
                multilanguage: null
            },
            singed_saved_copy_url:
                'https://hghltd.yandex.net/yandbtm?tld=ru&la=&text=%D0%BF%D1%80%D0%BE%D0%B1%D0%BA%D0%B8%20%D0%B2%20%D0%BC%D0%BE%D1%81%D0%BA%D0%B2%D0%B5&url=https%3A%2F%2Fyandex.ru%2Fmaps%2F213%2Fmoscow%2Fprobki%3Fsource%3Dtraffic&fmode=inject&tm=1488887337&l10n=ru',
            saveddoc_id: null,
            search_info: null,
            server_descr: 'TRAFFIC',
            server_region: '',
            shard: null,
            size: 0,
            snippets: {
                full: {
                    applicable: 1,
                    counter_prefix: '/snippet/traffic/',
                    data: {},
                    i18n_language: 'ru',
                    region: {
                        __is_plain: 1,
                        __package: 'YxWeb::Util::Region',
                        id: 213,
                        name: {
                            ru: {
                                genitive: 'Москвы',
                                locative: 'Москве',
                                nominative: 'Москва'
                            }
                        },
                        path: [10000, 10001, 225, 3, 1],
                        preposition: {
                            be: 'у',
                            en: 'in',
                            id: 'in',
                            kk: '',
                            ru: 'в',
                            tr: '',
                            tt: '',
                            uk: 'у',
                            uz: 'in'
                        },
                        tld: 'ru',
                        yandex_tld: 'ru'
                    },
                    region_ll: [37.620393, 55.75396],
                    region_name: 'Москва',
                    region_spn: [0.641442, 0.466439],
                    serp_info: {
                        flat: true,
                        format: 'json',
                        slot: 'full',
                        type: 'traffic'
                    },
                    slot: 'full',
                    slot_rank: 0,
                    template: 'traffic',
                    traffic: {
                        jams: [
                            {
                                time: 15,
                                val: '6'
                            },
                            {
                                time: 16,
                                val: '6'
                            },
                            {
                                time: 17,
                                val: '6'
                            },
                            {
                                time: 18,
                                val: '7'
                            },
                            {
                                time: 19,
                                val: '8'
                            },
                            {
                                time: 20,
                                val: '6'
                            }
                        ],
                        level: opts.level,
                        region_id: 213,
                        s2: '0',
                        semaphore: opts.semaphore,
                        text: 'Движение затруднено',
                        text_be: 'Рухацца цяжка',
                        text_en: 'Heavy traffic',
                        text_kk: 'Қозғалыс қиыншылықты',
                        text_ru: 'Движение затруднено',
                        text_tr: 'Çok yoğun trafik',
                        text_uk: 'Рух ускладнений',
                        timestamp: 1488887081,
                        url: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
                        url_new_style: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
                        url_new_style_ua: 'https://yandex.ru/maps/213/moscow/probki?source=traffic'
                    },
                    type: 'traffic',
                    types: {
                        all: ['snippets', 'traffic'],
                        kind: 'wizard',
                        main: 'traffic'
                    },
                    voiceInfo: {
                        ru: [
                            {
                                lang: 'ru-Ru',
                                text: 'Загруженность дорог  Москве составляет 6 баллов'
                            }
                        ]
                    }
                }
            },
            specInfo: null,
            stat: {
                p: 'p0',
                sn: 'traffic'
            },
            supplementary: {
                adresa: [
                    {
                        applicable: 0,
                        type: 'adresa',
                        types: ['snippets', 'adresa']
                    }
                ],
                domain_link: [
                    {
                        domain_href: 'https://yandex.ru/',
                        green_domain: 'https://yandex.ru',
                        green_tail: '…/probki?source=traffic'
                    }
                ],
                generic: [
                    {
                        applicable: 1,
                        attrs: {},
                        by_link: '',
                        counter_prefix: '/snippet/generic/',
                        headline: '',
                        headline_src: null,
                        is_generic: 1,
                        links: {},
                        passage_attrs: [],
                        passages: [],
                        template: 'generic',
                        type: 'generic',
                        types: {
                            all: ['snippets', 'generic'],
                            extra: [],
                            kind: 'snippets',
                            main: 'generic'
                        }
                    }
                ]
            },
            tier_name: '',
            torrent_click_share: null,
            torrent_query_share: null,
            url: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
            url_parts: {
                __is_plain: 1,
                __package: 'YxWeb::Util::Url',
                anchor: null,
                canonical: 'https://yandex.ru/maps/213/moscow/probki?source=traffic',
                cut_www: null,
                hostname: 'yandex.ru',
                link: '/maps/213/moscow/probki?source=traffic',
                path: '/maps/213/moscow/probki',
                port: null,
                query_string: 'source=traffic',
                scheme: 'https'
            }
        }
    };
};
