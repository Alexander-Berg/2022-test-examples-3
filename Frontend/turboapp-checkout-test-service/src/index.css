body {
    margin: 10px 20px;

    background: #f6f5f3;
}
