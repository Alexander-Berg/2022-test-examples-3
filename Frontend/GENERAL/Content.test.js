import React from 'react';
import { render } from 'enzyme';

import { SCOPE_PREFIX } from 'tools-access-react-redux-router/src/configs';

import { withRedux } from '~/src/common/hoc';
import { configureStore } from '~/src/abc/react/redux/store';

import Content from './Content';

jest.mock('../../PerfectionIssueGroup/PerfectionIssueGroup');
jest.mock('../../PerfectionExecution/PerfectionExecution');

const issues = [{
    canBeAppealed: false,
    description: { ru: 'Руководитель текущего сервиса находится в разных службах с руководителем вышестоящего сервис.' },
    execution: {
        name: { ru: 'Сервис перейдет в статус Требуется информация' },
        applyDate: new Date('2019-09-17T00:15:16Z'),
    },
    id: 1661,
    onReview: false,
    name: { ru: 'Руководитель вне кадровой иерархии' },
    recommendation: { ru: 'Назначьте другого руководителя или переместите сервис в нужное место по иерархии.' },
    weight: 10,
}];

const complaints = {
    count: 2,
    totalPages: 1,
    results: [
        {
            createdAt: new Date('2019-09-11T14:11:25.055800Z'),
            message: 'Problem1',
        },
        {
            createdAt: new Date('2019-09-11T14:30:47.575142Z'),
            message: 'Problem2',
        },
    ],
};

const paginatedComplaints = {
    count: 40,
    totalPages: 20,
    results: [
        {
            createdAt: new Date('2019-09-11T14:11:25.055800Z'),
            message: 'Problem1',
        },
        {
            createdAt: new Date('2019-09-11T14:30:47.575142Z'),
            message: 'Problem2',
        },
    ],
};

const issueGroup = {
    name: { ru: 'Понятность' },
    description: { ru: 'Сервис находится в желтой зоне по количеству проблем в группе Команда' },
    recommendation: { ru: 'Исправьте проблемы в сервисе и не допускайте новыx' },
    summary: {
        currentLevel: 'warning',
        currentWeight: 55,
        criticalWeight: 33,
        warningWeight: 66,
        currentExecution: {
            name: { ru: 'Сервис перейдёт в статус Требуется информация' },
            applyDate: new Date('2019-10-17T00:15:16Z'),
            isCritical: true,
        },
        nextExecution: {
            name: { ru: 'Сервис перейдет в красную зону при достижении порога 33% и будет Закрыт' },
            applyDate: new Date('2019-09-17T00:15:16Z'),
            isCritical: true,
        },
    },
};

describe('PerfectionDetails', () => {
    it('with issues content', () => {
        const wrapper = render(
            <Content
                type="Issues"
                values={issues}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('with empty content', () => {
        const wrapper = render(
            <Content
                type="Empty"
            />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('with complaints content', () => {
        const wrapper = render(
            <Content
                type="Complaints"
                values={complaints}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('with paginated complaints content', () => {
        const store = configureStore({
            initialState: {
                [SCOPE_PREFIX]: '',
            },
            fetcherOptions: {
                fetch: () => Promise.resolve(),
            },
        });

        const ContentConnected = withRedux(Content, store);

        const wrapper = render(
            <ContentConnected
                type="Complaints"
                values={paginatedComplaints}
                issuesPage={1}
                queryObj={{}}
                updateQueryStr={jest.fn()}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('with empty complaints content', () => {
        const wrapper = render(
            <Content
                type="Complaints"
                values={{ results: [] }}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('with issues content + issue groups', () => {
        const wrapper = render(
            <Content
                type="Issues"
                values={issues}
                issueGroup={issueGroup}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });
});
