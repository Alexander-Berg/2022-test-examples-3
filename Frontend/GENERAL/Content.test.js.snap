// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PerfectionDetails with complaints content 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Complaints"
>
  <div
    class="ComplaintItem"
  >
    <div
      class="ComplaintItem-Message"
    >
      Problem1
    </div>
    <div
      class="ComplaintItem-Data"
    >
      11 сент. 2019
    </div>
  </div>
  <div
    class="ComplaintItem"
  >
    <div
      class="ComplaintItem-Message"
    >
      Problem2
    </div>
    <div
      class="ComplaintItem-Data"
    >
      11 сент. 2019
    </div>
  </div>
</div>
`;

exports[`PerfectionDetails with empty complaints content 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Complaints"
>
  <div
    class="PerfectionDetails-EmptyImage"
  />
  <h3
    class="PerfectionDetails-EmptyHeader"
  >
    i18nRaw:complaintsNotFoundTitle
  </h3>
</div>
`;

exports[`PerfectionDetails with empty content 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Empty"
>
  <div
    class="PerfectionDetails-EmptyImage"
  />
  <h3
    class="PerfectionDetails-EmptyHeader"
  >
    i18nRaw:problemsNotFoundTitle
  </h3>
  <p
    class="PerfectionDetails-Message"
  >
    i18nRaw:problemsNotFound
    <a
      href="https://wiki.yandex-team.ru/Intranet/abc/features/suspicious/"
      target="_blank"
    >
      i18nRaw:problemsNotFoundLinkText
    </a>
  </p>
</div>
`;

exports[`PerfectionDetails with issues content + issue groups 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Issues"
>
  {"component":"PerfectionIssueGroup","props":{"className":"PerfectionDetails-ContentItem","issueGroup":{"name":{"ru":"Понятность"},"description":{"ru":"Сервис находится в желтой зоне по количеству проблем в группе Команда"},"recommendation":{"ru":"Исправьте проблемы в сервисе и не допускайте новыx"},"summary":{"currentLevel":"warning","currentWeight":55,"criticalWeight":33,"warningWeight":66,"currentExecution":{"name":{"ru":"Сервис перейдёт в статус Требуется информация"},"applyDate":"2019-10-17T00:15:16.000Z","isCritical":true},"nextExecution":{"name":{"ru":"Сервис перейдет в красную зону при достижении порога 33% и будет Закрыт"},"applyDate":"2019-09-17T00:15:16.000Z","isCritical":true}}}}}
  <div
    class="PerfectionIssue PerfectionDetails-ContentItem"
  >
    <h3
      class="PerfectionIssue-Title"
    >
      <span
        class="PerfectionIssueLevelIcon PerfectionIssueLevelIcon_type_small PerfectionIssueLevelIcon_level_warning PerfectionIssue-LevelIcon"
      />
      Руководитель вне кадровой иерархии
      <span
        class="PerfectionIssue-Weight"
      >
        10%
      </span>
    </h3>
    <div
      class="WikiFormatter PerfectionIssue-Description"
    />
    <div
      class="WikiFormatter PerfectionIssue-Recommendation"
    />
    {"component":"PerfectionExecution","props":{"className":"PerfectionIssue-Execution","execution":{"name":{"ru":"Сервис перейдет в статус Требуется информация"},"applyDate":"2019-09-17T00:15:16.000Z"}}}
  </div>
</div>
`;

exports[`PerfectionDetails with issues content 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Issues"
>
  <div
    class="PerfectionIssue PerfectionDetails-ContentItem"
  >
    <h3
      class="PerfectionIssue-Title"
    >
      <span
        class="PerfectionIssueLevelIcon PerfectionIssueLevelIcon_type_small PerfectionIssueLevelIcon_level_warning PerfectionIssue-LevelIcon"
      />
      Руководитель вне кадровой иерархии
      <span
        class="PerfectionIssue-Weight"
      >
        10%
      </span>
    </h3>
    <div
      class="WikiFormatter PerfectionIssue-Description"
    />
    <div
      class="WikiFormatter PerfectionIssue-Recommendation"
    />
    {"component":"PerfectionExecution","props":{"className":"PerfectionIssue-Execution","execution":{"name":{"ru":"Сервис перейдет в статус Требуется информация"},"applyDate":"2019-09-17T00:15:16.000Z"}}}
  </div>
</div>
`;

exports[`PerfectionDetails with paginated complaints content 1`] = `
<div
  class="PerfectionDetails-Content PerfectionDetails-Content_Type_Complaints"
>
  <div
    class="ComplaintItem"
  >
    <div
      class="ComplaintItem-Message"
    >
      Problem1
    </div>
    <div
      class="ComplaintItem-Data"
    >
      11 сент. 2019
    </div>
  </div>
  <div
    class="ComplaintItem"
  >
    <div
      class="ComplaintItem-Message"
    >
      Problem2
    </div>
    <div
      class="ComplaintItem-Data"
    >
      11 сент. 2019
    </div>
  </div>
  <div
    class="AbcPagination AbcPagination_type_pages AbcPagination_type_pages PerfectionDetails-Pagination"
  >
    <ul
      class="Pagination"
    >
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_current Pagination-Inner_kind_page"
          href="?issuesPage=1"
        >
          1
        </a>
      </li>
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_kind_page"
          href="?issuesPage=2"
        >
          2
        </a>
      </li>
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_kind_page"
          href="?issuesPage=3"
        >
          3
        </a>
      </li>
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_kind_page"
          href="?issuesPage=4"
        >
          4
        </a>
      </li>
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_kind_page"
          href="?issuesPage=5"
        >
          5
        </a>
      </li>
      <li
        class="Pagination-Item"
      >
        <a
          class="Link Link_theme_black AbcPagination-Item Pagination-Inner Pagination-Inner_kind_text"
          href="?issuesPage=2"
        >
          i18n:next
        </a>
      </li>
    </ul>
  </div>
</div>
`;
